src/core/internal/arrayop.d

```diff
@@ -183,6 +194,9 @@ bool isBinaryAssignOp(string op)
     return op.length == 2 && op[1] == '=' && isBinaryOp(op[0 .. 1]);
 }
 
+// Generate mixin expression to perform scalar arrayOp loop expression, assumes
+// `pos` to be the current slice index, `args` to contain operand values, and
+// `res` the target slice.
```

Comment:
I don't see pos, args, or res in the parameter list or even in the function body. Also, when documenting parameters, please use Ddoc conventions, i.e. a `Params:` block.